test/framework/kubernetes.go

```diff
@@ -133,3 +134,31 @@ func loadResource(namespace, uri string, resourceRef meta.ResourceObject, before
 	}
 	return nil
 }
+
+// WaitForPodsToContainTextInLog waits for pods of specified deployment config to contain specified text in log
+func WaitForPodsToContainTextInLog(namespace, dcName, logText string, timeoutInMin int) error {
+	return WaitFor(namespace, fmt.Sprintf("Pods for deployment config '%s' contain text '%s'", dcName, logText), time.Duration(timeoutInMin)*time.Minute, func() (bool, error) {
+		pods, err := GetPodsWithLabels(namespace, map[string]string{"deploymentconfig": dcName})
+		if err != nil {
+			return false, err
+		}
+
+		// Container name is equal to deployment config name
+		return checkPodsContainingTextInLog(namespace, pods, dcName, logText)
+	})
+}
+
+func checkPodsContainingTextInLog(namespace string, pods *corev1.PodList, containerName, text string) (bool, error) {
+	for _, pod := range pods.Items {
+		containsText, err := isPodContainingTextInLog(namespace, &pod, containerName, text)
+		if err != nil || !containsText {
```

Comment:
small mistake here I think
you should stop looking into pods if you find something or ?

Reply:
Actually not. The purpose is to check all pods than they contain required text.
If test is found then another pod is inspected (for cycle continue), if text is not found then false is returned from the method.

Reply:
Can we then call the method `*AllPods*` ? would be clearer I think

Reply:
Done